libs/jitsu-js/src/destination-plugins/gtm.ts

```diff
@@ -7,51 +7,92 @@ export type GtmDestinationCredentials = {
   dataLayerName?: string;
 } & CommonDestinationCredentials;
 
+function omit(obj: any, ...keys: string[]) {
+  const set = new Set(keys);
```

Comment:
on small number of elements Set is actually slower than arrays. So simple keys.includes will do